styles/pov.h

```diff
@@ -4,14 +4,48 @@
 // TODO(hubbe): Make it work with FastLED
 #if defined(ENABLE_WS2811)
 
-// Usage: &style_pov or StylePtr<StylePtr>
-// return value: suitable for preset array
-
-// This style draws images in the air if you swing the saber
-// carefully back and forth. Unfortunately you can't really see
-// the result without a slow exposure camera. It's possible that
-// dotstar APA102 pixels would be fast enough to do the effect
-// full justice, but I haven't tried that.
+/* 
+This style draws images in the air if you swing the saber
+carefully back and forth. Unfortunately you can't really see
+the result without a slow exposure camera. It's possible that
+dotstar APA102 pixels would be fast enough to do the effect
+full justice, but I haven't tried that.
+
+Usage: &style_pov
+or: StylePtr<StylePOV<>>()
+or: StylePtr<StylePOV<MIN_DEGREES, MAX_DEGREES, REPEAT, MIRROR>>()
+MIN_DEGREES: integer (default -45)
+MAX_DEGREES: integer (default 45)
+REPEAT: bool (default false)
+MIRROR: bool (default true)
+return value: COLOR
+Since MIN_DEGREES, MAX_DEGREES, REPEAT and MIRROR all have defaults, those arguments can be omitted.
+StylePOV<> draws one instance of the image in the top portion of the swing "window".
```

Comment:
We also need to tell people what the arguments actually do.

Reply:
I think this is done?